docroot/modules/humsci/hs_capx/src/Form/CapxCredsForm.php

```diff
@@ -128,6 +128,16 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
 
     // We have valid username and passwords, lets get the organization data.
     $this->capx->syncOrganizations();
+
+    // Enable capx migrations.
+    $this->configFactory->getEditable('migrate_plus.migration.hs_capx')
+      ->set('status', TRUE)
+      ->save();
+    $this->configFactory->getEditable('migrate_plus.migration.hs_capx_images')
```

Comment:
This config was removed.

Reply:
since we do `cim --partial` on acquia, we gotta explicitly delete any configs via update hooks